companies-manifest.js

```diff
@@ -853,6 +853,40 @@ const companies = [
       },
     },
   },
+  {
+    key: 'https://wazirx.com/',
+    similarWebGlobalRank: 48011,
+    name: 'WazirX',
+    logo: null,
```

Comment:
Do you have the logo for the company?

Reply:
The logo provided is a png. Do I have to get an SVG one?